oracle-ebs/5.2/modules/ROOT/pages/oracle-ebs-connector-reference.adoc

```diff
@@ -0,0 +1,2656 @@
+= Oracle EBS 12.1 Connector Reference
+:page-aliases: connectors::oracle/oracle-ebs-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Oracle EBS 12.1 Connector Version 5.2
```

Comment:
```suggestion
Anypoint Connector for Oracle EBS 12.1 (Oracle EBS 12.1 Connector) Version 5.2
```

Reply:
Normally, we would remove this line but I think in this case it helps avoid the confusion caused by the fact 12.1 is part of the connector name. 
The title would normally have the connector version but in this case I think it would be confusing to have: Oracle EBS 12.1 Connector 5.2 Reference. 
What do you think?

Reply:
Hmm, not sure what you mean by connector version, but I think we should keep the 12.1 just because there is a 12.2 version as well so it would be more clear which docs the user is looking at. But if you mean the version 5.2, then yes I think we can remove that.